lectures/classes/README.md

```diff
@@ -38,15 +38,15 @@ Creating trivial types - give the class name then list the values to be assigned
 members, _in order_, inside braces:
 
 ```C++
-Complex mk_imaginary_unit(void) {
```

Comment:
The voids were added to make it explicit what is intended. Happy to change this for now but I think its a good best practice to do keep the void.

Reply:
I changed it as best practise for C++ seems to be not to include void: https://github.com/isocpp/CppCoreGuidelines/blob/master/CppCoreGuidelines.md#nl25-dont-use-void-as-an-argument-type

I believe there is technically a difference between foo() and foo(void) in C - the first being any number of arguments and the second being no arguments - but they should be treated the same by C++ compilers, and foo(void) is primarily only valid C++ to maintain compatibility with C code that uses it